cmd/process-agent/collector.go

```diff
@@ -491,7 +537,13 @@ func (l *Collector) consumePayloads(results *api.WeightedQueue, fwd forwarder.Fo
 			case checks.Pod.Name():
 				// Orchestrator intake response does not change RT checks enablement or interval
 				updateRTStatus = false
-				responses, err = fwd.SubmitOrchestratorChecks(forwarderPayload, payload.headers, int(orchestrator.K8sPod))
+
+				// Manifests don't need this header
+				if payload.headers.Get(headers.ContainerCountHeader) == "" {
```

Comment:
We could either use one of the header or check name here to submit manifests and pod check messages separately. Feel free to tell me what's your preference

Reply:
I update the check name instead of add another check to avoid fetching pod twice times